code/CSS Responsive Assignment/style.css

```diff
@@ -0,0 +1,97 @@
+
+body {
+    margin: 0;
+}
+
+#profile {
+    background-color: #f9b8b6;
+    padding: 1em;
+    margin: 1em;
+}
+
+#profile > img {
+    outline: 0.3rem solid #fff;
+    margin: 0.5em 0;
+}
+
+#profile > span {
+    font-size: 1.3em;
+    font-weight: 600;
+    padding-bottom: 1em;
+}
+
+.card {
+    border: 3px solid #006d9e;
+    margin: 0.2rem;
+}
+
+img {
+    width: 100%;
+}
+
+header {
+    background-color: #a7cddf;
+    font-size: 2em;
+    display: flex;
+    justify-content: center;
+}
+
+@media screen and (min-width: 500px){
+    .card{
+        display:flex;
+        flex-direction: row;
+    }
+    .card.right{
+        flex-direction:row-reverse;
+    }
+    .card img{
+        height: auto;
+        width: 80%;
+    }
+    .photo-label{
+        padding:15px;
+    }
+}
+
+@media screen and (min-width: 750px){
+    
+    #gallery{
+        display:flex;
+        flex-direction:row;
+        justify-content:flex-start;
+        flex-wrap:wrap;
+        justify-content:center;
+
+    }
+
+
+    .card{
+        display:flex;
+        flex-direction:column;
+        flex-grow:1;
+        width:30%;
+
+    }
+
+    .card.right{
+        flex-direction:column;
+    }
+    .card img{
+        width: auto;
+        height: 100%;

```

Comment:
I think you must have added this line by accident after you finished the assignment. Everything looked great once I took this line out.